### ISSUE TITLE
CliRenderer fails with ErrorException when "exec" is disabled

### ISSUE THREAD

**Issue by element-code** [kint-php/kint#350]
Trying to `dd` in the CLI results in an `ErrorException`.
```txt
Type:        ErrorException
Message:     exec() has been disabled for security reasons
Filename:    /vendor/kint-php/kint/src/Renderer/CliRenderer.php
Line Number: 76
Backtrace:
-76 - /vendor/kint-php/kint/src/Renderer/CliRenderer.php::exec
- 345 - /vendor/kint-php/kint/src/Kint.php::__construct
- 570 - /vendor/kint-php/kint/src/Kint.php::createFromStatics
- 384 - /vendor/codeigniter4/framework/system/Common.php::dump
- 25 - /app/Commands/DeployAddUser.php::dd
```

**Comment by jnvsor** [comment]
Hmm, that's a suphp plugin or something isn't it? I'll throw an all-purpose try/catch around it but it's not going to be guaranteed to fix every plugin that tries to disable `exec`

**Comment by jnvsor** [comment]
This should be fixed in 3d92c5e92d561613028836ea7bf6c4e2b8c27a08 - can you confirm that for me @element-code ?

**Comment by element-code** [comment]
Yes, confirmed
